tests/acceptance/interactions-test.js

```diff
@@ -156,3 +157,17 @@ test('#fillInByName fills input by name', (assert) => {
     assert.equal(val, targetValue, 'expect the input to have the target value');
   });
 });
+
+test('#chooseFromDropdown selects a dropdown option by option', (assert) => {
```

Comment:
Echoing what @romina is suggesting – I would even change the test name to say: "...dropdown option by option *text*".